docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+# This file is part of stable-diffusion-webui (https://github.com/sd-webui/stable-diffusion-webui/).
+
+# Copyright 2022 sd-webui team.
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+
+version: '3.3'
+
+services:
+  stable-diffusion:
+    container_name: sd-webui
+    image: hlky/stable-diffusion-webui:dev
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      # Enable local development
+      # - .:/sd
+      # Make it easier to exfiltrate your outputs
+      - ./outputs:/sd/outputs
+      # Cache models and stuff in a volume so they're long-lived
+      - user_data:/sd/user_data
```

Comment:
I think we should put both of these into a single folder that can be mounted so this works in places where multiple mounts aren't supported (like runpod).  A user can choose to mount `outputs` separate from `model_weights`, or they could mount them both together.

Reply:
Runpod does not use the docker compose file, so I'm not sure I understand. Exfiltration of generations on Runpod _is_ something that should be addressed, though. I'm not sure there's a good story around it right now.